### ISSUE TITLE
Wrong version of selenium is used

### ISSUE THREAD

**Issue by mpowaga** [xolvio/chimp#252]
```
$ chimp --ddp=http://localhost:3000 --browser=chrome --path=tests --mocha --debug

[chimp] Running...
[chimp] interrupting
[chimp] no processes to interrupt
[chimp][selenium] Selenium object created
[chimp][selenium] Start
[chimp][selenium] Installing Selenium + drivers if needed
selenium-standalone installation [====================] 100% 0.0s
[chimp][selenium] hub can be seen at http://localhost:63343/wd/hub
[chimp] Finished running async processes with errors
[chimp] run complete with errors [Error: Missing /Users/Michal/.nvm/versions/node/v5.2.0/lib/node_modules/chimp/node_modules/selenium-standalone/.selenium/selenium-server/2.50.1-server.jar]
[Error: Missing /Users/Michal/.nvm/versions/node/v5.2.0/lib/node_modules/chimp/node_modules/selenium-standalone/.selenium/selenium-server/2.50.1-server.jar]
Error in chimp init [Error: Missing /Users/Michal/.nvm/versions/node/v5.2.0/lib/node_modules/chimp/node_modules/selenium-standalone/.selenium/selenium-server/2.50.1-server.jar]
```

I solved this by manually install latest selenium:

```
$ npm install selenium-standalone -g
$ selenium-standalone install --version=2.50.1 --baseURL=https://selenium-release.storage.googleapis.com --basePath=`npm config get prefix`/lib/node_modules/chimp/node_modules/selenium-standalone/.selenium
```

**Comment by enaros** [comment]
had the same problem, thanks @mpowaga for this quick fix!

**Comment by samhatoum** [comment]
Thank you for reporting, will look into it.

**Comment by Sanjo** [comment]
Fixed with 0.26.0
